spectaql-config.yml

```diff
@@ -0,0 +1,20 @@
+spectaql:
```

Comment:
should this file be in dev folder? (like otel-agent-config.yaml)

Reply:
The dev folder seems more like a place to keep files involved in the docker-compose dev flow. Not sure if moving the file there makes sense. Wdyt @bodymindarts ?